code/game/objects/items/weapons/grenades/spawnergrenade.dm

```diff
@@ -18,10 +18,8 @@
 		for(var/mob/living/carbon/C in viewers(T, null))
 			C.flash_eyes()
 
-		for(var/i=1, i<=deliveryamt, i++)
-			var/atom/movable/x = new spawner_type
-			x.admin_spawned = admin_spawned
```

Comment:
I removed this because- it didn't really make much sense. its not admin spawned?

Reply:
Presumably it takes the grenade's `admin_spawned` variable, so I would keep that in

Reply:
Ohhh that makes more sense now

Reply:
reverted